src/test/java/lux/IndexTest.java

```diff
@@ -198,8 +198,8 @@ public void testStoreBinary () throws Exception {
         indexer.storeDocument(indexWriter, "/lux/compiler/test-module.xqy", 
                 getClass().getClassLoader().getResourceAsStream("lux/compiler/test-module.xqy"));
         indexWriter.close();
-        System.out.println 
-             (String.format("indexed path-values for test-module.xqy in %d bytes", dir.sizeInBytes()));
+//        System.out.println
```

Comment:
Delete these commented-out sysouts. They never should have been there to begin with.